tests/deploy_test_anker.py

```diff
@@ -0,0 +1,281 @@
+)
+
+solido_address = result['solido_address']
+st_sol_mint_address = result['st_sol_mint_address']
+print(f'> Created instance at {solido_address}')
+
+approve_and_execute = get_approve_and_execute(
+    multisig_program_id=multisig_program_id,
+    multisig_instance=multisig_instance,
+    signer_keypair_paths=[maintainer.keypair_path],
+)
+
+print('\nAdding maintainer ...')
+transaction_result = solido(
+    'add-maintainer',
+    '--multisig-program-id',
+    multisig_program_id,
+    '--solido-program-id',
+    solido_program_id,
+    '--solido-address',
+    solido_address,
+    '--maintainer-address',
+    maintainer.pubkey,
+    '--multisig-address',
+    multisig_instance,
+    keypair_path=maintainer.keypair_path,
+)
+approve_and_execute(transaction_result['transaction_address'])
+print(f'> Maintainer {maintainer.pubkey} added.')
+
+# Next up is the token pool, but to be able to set that up,
+# we need some stSOL (and some UST, which we have already),
+# and then we need to put that in some new accounts that the
+# pool will take ownership of.
+print('\nSetting up stSOL-UST pool ...')
+solido(
+    'deposit',
+    '--solido-program-id',
+    solido_program_id,
+    '--solido-address',
+    solido_address,
+    '--amount-sol',
+    '0.1',
+)
+pool_ust_keypair = create_test_account(f'{test_dir}/pool-ust.json', fund=False)
+pool_st_sol_keypair = create_test_account(f'{test_dir}/pool-st-sol.json', fund=False)
+spl_token('create-account', ust_mint_address, pool_ust_keypair.keypair_path)
+spl_token('create-account', st_sol_mint_address, pool_st_sol_keypair.keypair_path)
+spl_token('transfer', ust_mint_address, '0.1', pool_ust_keypair.pubkey)
+spl_token('transfer', st_sol_mint_address, '0.1', pool_st_sol_keypair.pubkey)
+result = solido(
+    'anker',
+    'create-token-pool',
+    '--token-swap-program-id',
+    token_swap_program_id,
+    '--ust-mint-address',
+    ust_mint_address,
+    '--ust-account',
+    pool_ust_keypair.pubkey,
+    '--st-sol-account',
+    pool_st_sol_keypair.pubkey,
+)
+token_pool_address = result['pool_address']
+print(f'Pool address is {token_pool_address}.')
+
+print('\nCreating Anker instance ...')
+result = solido(
+    'anker',
+    'create',
+    '--solido-program-id',
+    solido_program_id,
+    '--solido-address',
+    solido_address,
+    '--anker-program-id',
+    anker_program_id,
+    '--ust-mint-address',
+    ust_mint_address,
+    '--token-swap-pool',
+    token_pool_address,
+    '--wormhole-core-bridge-program-id',
+    DEVNET_WORMHOLE_CORE_BRIDGE_PROGRAM_ID,
+    '--wormhole-token-bridge-program-id',
+    DEVNET_WORMHOLE_TOKEN_BRIDGE_PROGRAM_ID,
+    '--terra-rewards-address',
+    DEVNET_TERRA_REWARDS_ADDRESS,
+)
+anker_address = result['anker_address']
+print(f'> Created instance at {anker_address}.')
+
+sol_balance_post = float(solana('balance').split(' ')[0])
+total_cost_sol = sol_balance_pre - sol_balance_post
+print(f'\nDeployment cost {total_cost_sol:.3f} SOL.')
```

Comment:
Would be nice to see how much each of the Anker, multisig, and Solido deployments cost.

Reply:
Would be nice indeed. I intended to still add it for a long time, but this PR has been sitting here for a long time, I’ll just merge it now without this feature.